worker/src/RTC/RtpDictionaries/RtpEncodingParameters.cpp

```diff
@@ -151,9 +151,6 @@ namespace RTC
 		if (!this->scalabilityMode.empty())
 		{
 			jsonObject["scalabilityMode"] = this->scalabilityMode;
-			jsonObject["spatialLayers"]   = this->spatialLayers;
-			jsonObject["temporalLayers"]  = this->temporalLayers;
-			jsonObject["ksvc"]            = this->ksvc;
```

Comment:
Can we please keep those? We don't need to remove them from `dump()` method whose output is not even documented (private API just for debugging). We really need them for debugging.

Reply:
I'm gonna do it

Reply:
done here: https://github.com/versatica/mediasoup/pull/518/commits/b2047f73512197086ec1129098c4be8e6fdf8e2b